controllers/hyperconverged/hyperconverged_controller.go

```diff
@@ -476,6 +480,33 @@ func (r *ReconcileHyperConverged) handleUpgrade(req *common.HcoRequest, init boo
 	return nil, nil
 }
 
+func (r *ReconcileHyperConverged) handleNMO(req *common.HcoRequest) error {
+	oldNmoCRs := &nmoapioldv1beta1.NodeMaintenanceList{}
+	err := r.client.List(req.Ctx, oldNmoCRs)
+	if err != nil {
+		return err
+	}
+
+	for _, oldNmoCr := range oldNmoCRs.Items {
+		newNmoCr := &nmoapiv1beta1.NodeMaintenance{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: oldNmoCr.Name,
+			},
+			Spec: nmoapiv1beta1.NodeMaintenanceSpec{
+				NodeName: oldNmoCr.Spec.NodeName,
+				Reason:   oldNmoCr.Spec.Reason,
+			},
+		}
+		err := r.client.Create(req.Ctx, newNmoCr)
+		if err != nil && !strings.Contains(err.Error(), "already exists") {
```

Comment:
can you please use IsAlreadyExists(err) from `k8s.io/apimachinery/pkg/api/errors`

Reply:
fixed, thanks.